sdk/hdinsight/Microsoft.Azure.HDInsight.Job/CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
-## 3.0.0-preview.1 (Unreleased)
-
+## 3.0.0-preview.2 (Unreleased)
```

Comment:
Add `Release History` above this line.  Please refer other data plane sdk about the CHANGELOG.md